ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/NETWORK_PORTS_TESTS.yml

```diff
@@ -141,6 +141,25 @@ network_ports:
       channel_id: 43
       mode: "active"
 
+  # The next two switches are targeted using a like regex filtering on correct platform type
+  # Test regex and platform match criteria
+  - switches:
+      - network-ports-tests.*
```

Comment:
Since all the nodes are matching this regex, we can add a new node not matching this regex and having platform 720XPM-48Y6.

Reply:
Yes, good test! Expected result that if it does not match regex, logic will skip check for match platform, config is not generated for the ports. In other words, if both regex and platform are supplied, both must match to generate configurations.